client/blocks/login/login-form.scss

```diff
@@ -65,12 +65,3 @@ form {
     	}
     }
 }
-
-.wp-login__main .login.is-akismet .is-social-first .card.login__form > .social-buttons__button.button {
```

Comment:
Moved to https://github.com/Automattic/wp-calypso/pull/100807/files#diff-934caa216b18163d15b19bde2bf9c56db6984aba9bed8cd927226eb56cedee12R160